tests/src/tests.rs

```diff
@@ -421,6 +444,23 @@ fn test(
     ok
 }
 
+fn update_image(png_path: &Path, ref_path: &Path) {
+    let Output { status, stdout, stderr } = Command::new("oxipng")
```

Comment:
I should probably have been more clear before, but since we're in Rust code here it might be nice to depend on `oxipng` as a crate instead so that there is no system dependency.

Reply:
Great point, done.